src/cluster_legacy.c

```diff
@@ -5282,6 +5283,20 @@ void clusterCloseAllSlots(void) {
  * Cluster state evaluation function
  * -------------------------------------------------------------------------- */
 
+void clusterLogWhyFail(int reason) {
+    char *msg;
+    switch (reason) {
+    case CLUSTER_FAIL_NOT_FULL_COVERAGE:
+        msg = "Detect there is at least a hash slot uncovered (no available node is serving it). "
+              "Please check the 'cluster-require-full-coverage' configuration option.";
+        break;
+    case CLUSTER_FAIL_MINORITY_PARTITION: msg = "In a minority partition."; break;
+    default: msg = "Unknown reason code."; break;
+    }
+    serverLog(LL_NOTICE, "Currently cluster unable to work: %s", msg);
```

Comment:
Let's log errors at `WARNING`
```suggestion
    serverLog(LL_WARNING, "Cluster is currently down: %s", msg);
```